### PR TITLE
Merge Changes from Maven and Linkinsoldier into Jojo Repo

### PR DESCRIPTION
**Pull request by Linkinsoldier**
As discussed on kodinerds.net we should first merge our changes, then create a pull request and then get the new release into the kodinerds repo

### CODE REVIEW

File: navigation.py [hunk 1]
```diff
@@ -385,6 +624,55 @@ def listPage(page_id):
...
+def getTMDBDataFromCache(title, year = None, attempt = 1, content='movie'):
+    return TMDBCache.cacheFunction(getTMDBData, title, year, attempt, content)
+
+def getTMDBData(title, year=None, attempt = 1, content='movie'):
+    rating = None
+    poster_path = None
+    tmdb_id = None
+    splitter = [' - ', ': ', ', ']
+    tmdb_api = base64.b64decode('YTAwYzUzOTU0M2JlMGIwODE4YmMxOTRhN2JkOTVlYTU=') # ApiKey Linkinsoldier
+    # tmdb_api = base64.b64decode('YjM0NDkwYzA1NmYwZGQ5ZTNlYzlhZjIxNjdhNzMxZjQ=') # ApiKey from Sandman's Amazon VOD Addon
```
Comment (trummerjo) [inline navigation.py]: please remove!

File: skygo.py [hunk 2]
```diff
@@ -213,17 +274,22 @@ def play(self, manifest_url, package_code, info_tag=None, apix_id=None):
                 li = xbmcgui.ListItem(path=manifest_url)
                 if info_tag:
                     li.setInfo('video', info_tag)
-                # Force smoothsteam addon
-                li.setProperty('inputstream.smoothstream.license_type', 'com.widevine.alpha')
+                # Inputstream settings
+                is_addon = getInputstreamAddon()
```
Comment (trummerjo) [inline skygo.py]: Could be removed.
inputstream.smoothstream is actually obsolete and we could always use inputstream.adaptive

**Review by trummerjo — CHANGES_REQUESTED** [review]
Please check §3. Attribution from:
https://www.themoviedb.org/documentation/api/terms-of-use

File: navigation.py (hunk 1 above)
Comment (trummerjo) [reply navigation.py]: `#tmdb_api = base64.b64decode('YjM0NDkwYzA1NmYwZGQ5ZTNlYzlhZjIxNjdhNzMxZjQ=') # ApiKey from Sandman's Amazon VOD Addon
 `
please remove this line

Comment (Linkinsoldier) [reply navigation.py]: removed and Comment for using TMDb Api added

**Pull request merged**